tests/jballerina-integration-test/src/test/resources/http/src/httpservices/33_disable_https_service.bal

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+
+http:ServiceEndpointConfiguration serviceConf = {
+    secureSocket: {
```

Comment:
Don't we need disable:true here?

Reply:
Disabling is only applicable to client side. So we don't need it here.